### ISSUE TITLE
Build fails with "DSO missing from command line" for libFRC_NetworkCommunication.so.17.0.0

### ISSUE THREAD

**Issue by pbfy0** [JacisNonsense/GradleRIO-C#3]
I'm trying to build my command-based robot code with GradleRIO-C. The build gets to the linking stage, then fails with

```
c:/users/paul/.gradle/gradlerioc/toolchain/windows/frc/bin/../lib/gcc/arm-frc-linux-gnueabi/4.9.3/../../../../arm-frc-linux-gnueabi/bin/ld.exe: C:\Users\Paul\Documents\GitHub\2017RobotGradle\build\dependencies\third\Toolsuite\cpp\lib\libCTRLib.a(CanTalonSRX.o): undefined reference to symbol 'FRC_NetworkCommunication_CANSessionMux_readStreamSession'

C:\Users\Paul\Documents\GitHub\2017RobotGradle\build/dependencies/wpi/Linux/arm\libFRC_NetworkCommunication.so.17.0.0: error adding symbols: DSO missing from command line
```
According to the [options.txt](https://github.com/Open-RIO/GradleRIO-C/files/948266/options.txt) file from linking, the module doesn't make any attempt to link the FRC_NetworkCommunication library. I don't really understand if it _should_ need to link that. However, if I manually add "-lFRC_NetworkCommunication" to the options.txt, it links successfully but produces a huge file (10 mb). I'm using the default quickstart project from github.

**Comment by JaciBrunning** [comment]
hmm. Modify your build.gradle to include `wpi { talonSrxVersion = "4.4.1.9" }` and report back if this problem still exists.

**Comment by yabberyabber** [comment]
I think I'm running into a similar issue.

When I use the Quickstart as provided, it compiles just fine.  However, if I change HelloRobot.cpp to contain the following:

```
#include "WPILib.h"
#include "HelloRobot.hpp"
#include "CANTalon.h"               //this line was added

#include <iostream>

int add(int a, int b) {
    return a + b;
}

class Robot: public SampleRobot {
public:
    Robot() { }

    void RobotInit() {
        CANTalon *a = new CANTalon(8);          //this line was added
        std::cout << "Hello World" << std::endl;
        std::cout << add(1, 2) << std::endl;
    }
};

START_ROBOT_CLASS(Robot)
```

I get the following error:
```
:linkMy_programExecutable
/usr/lib/x86_64-linux-gnu/gcc/arm-frc-linux-gnueabi/4.9.3/../../../../../arm-frc-linux-gnueabi/bin/ld: /home/andrew/Downloads/gradle_frc/build/dependencies/wpi/lib/libwpilibc.so: undefined reference to symbol '_ZN14ITableListener14ValueChangedExEP6ITableN4llvm9StringRefESt10shared_ptrIN2nt5ValueEEj'
/home/andrew/Downloads/gradle_frc/build/dependencies/wpi/Linux/arm/libntcore.so: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
```


I tried adding `wpi { talonSrxVersion = "4.4.1.9" }` into my build.gradle, but that gave me the same output.

Out of curiosity, I also tried adding `wpi { talonSrxVersion = "4.4.1.11" }` into my build.gradle.  When I built with that setup, I got the same exact error as pbfy0 above.

**Comment by JaciBrunning** [comment]
See #5 for a temporary work around. Sorry for the delay

**Comment by JaciBrunning** [comment]
This issue should now be fixed. Upgrade to GradleRIO `2017.3.1` for this to be fixed, and you can remove the #5 workaround in your build.gradle.
